python/lsst/synpipe/makeBlendedCat.py

```diff
@@ -1,10 +1,5 @@
-#!/usr/bin/env python
```

Comment:
Why does this go away and not in `python/lsst/synpipe/makeFakeGalaxy.py` too?

Reply:
I must have missed that one. I'll fix it and check for others.